deploy/osps/default/osp-amzn2.yaml

```diff
@@ -591,7 +594,15 @@ spec:
               }
 
               # get the default interface IP address
+              {{- if eq .NetworkIPFamily "IPv6" }}
```

Comment:
We set both IPv4,IPv6 in dual stack setup only. 
If IPFamily is IPv6 only IPv6 should be set.

Reply:
nm my eyes fooled me